thunder/executors/transformer_engineex.py

```diff
@@ -332,6 +337,12 @@ def _te_functional_linear_backward_meta(
 
 LINEAR_CALLS_COUNTER = 0
 
+if TE_AVAILABLE:
+    _DEFAULT_RECIPE = recipe.DelayedScaling()
+
+IMPORT_CTX_TE_KEY = "transformer_engine"
+FP8_RECIPE_KEY = "fp8_recipe"
```

Comment:
What if Thunder gets its own fp8_recipe? Should we call it "te_fp8_recipe" already now?

Reply:
Fair point, will update this. Thanks!